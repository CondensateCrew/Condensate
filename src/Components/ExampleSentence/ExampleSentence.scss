@import '../../css/variables.scss';
@import '../../css/index.scss';

.template-question-h3 {
  width: 40vw;
  padding: $large-space;
  box-shadow: $external-shadow;
  border-radius: $big-blocks-border-radius;
}