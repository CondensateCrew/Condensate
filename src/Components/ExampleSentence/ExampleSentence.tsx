import React from 'react';
import './ExampleSentence.scss';

interface Props {
  exampleSentence: string;
};

const ExampleSentence:React.FC<Props> = ({ exampleSentence }) => {
  return (
    <h3 className='template-question-h3'>{exampleSentence}</h3>
  )
}

export default ExampleSentence;
