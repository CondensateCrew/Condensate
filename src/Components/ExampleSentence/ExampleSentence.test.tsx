import React from 'react';
import ExampleSentence from './ExampleSentence';
import { mount } from 'enzyme';

describe('ExampleSentence Component', () => {
  it('should match the snapshot', () => {
    const wrapper = mount(<ExampleSentence
      exampleSentence={'A neighbor sings at a most ungodly hour'}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
