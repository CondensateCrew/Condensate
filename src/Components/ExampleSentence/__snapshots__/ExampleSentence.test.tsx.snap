// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExampleSentence Component should match the snapshot 1`] = `
<ExampleSentence
  exampleSentence="A neighbor sings at a most ungodly hour"
>
  <h3
    className="template-question-h3"
  >
    A neighbor sings at a most ungodly hour
  </h3>
</ExampleSentence>
`;
